src/scss/components/ec-grid/ec-grid.scss

```diff
@@ -0,0 +1,92 @@
+$gutters: 24px;
+$number-of-columns: 12;
+
+@mixin common {
+  display: flex;
+  justify-content: flex-start;
+  width: calc(100% + #{$gutters});
+  margin-left: calc(#{$gutters} / -2);
+}
+
+.ec-grid {
+  overflow: hidden;
```

Comment:
Curious to know why we want this behaviour?

Reply:
In order to have offsets and always full width I need to have negative margins which makes the ec-row moving outside the parent by "gutter px/ 2". I am hiding this with overflow. It's ain't elegant but for now it seem to work.

Reply:
What about adding padding-left: 12px and padding-right: 12px instead? I think we want to have a gap on the sides too.

Reply:
> I think we want to have a gap on the sides too.
I do agree with this.

Reply:
> > I think we want to have a gap on the sides too.
> 
> I do agree with this.

That's how I had it initially and with a modifier class I was making it full width but we said to have it permanently full width.